.github/PULL_REQUEST-TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+### JIRA link
+
+
+
+### Change description
+
+
+
+### Work checklist
+
+- [ ] Tests added where applicable
```

Comment:
Is a checklist for UI changes on an API really necessary? And the gobbledygook merge instructions at the bottom? I noticed a lot of APIs use something like https://github.com/companieshouse/acsp-api/blob/main/.github/PULL_REQUEST_TEMPLATE.md . Wondering if its worth tidying up this one?

Reply:
Changed the template to match @makhtar-ch request

Reply:
now the test line is missing

- [ ] Tests added where applicable

Reply:
line added

Reply:
The test line is in the PR checklist section. Now we have two.